Maths/AverageMean.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @function AverageMean
+ * @description This script will find the mean value of a array of numbers.
+ * @param {number[]} numbers - Array of numeric values
+ * @return {number} - mean of input numbers
+ * @see [Mean](https://en.wikipedia.org/wiki/Mean)
+ * @example AverageMean([1, 2, 4, 5]) = 3
+ * @example AverageMean([10, 40, 100, 20]) = 42.5
+ */
+
+export const AverageMean = (numbers: number[]): number => {
+  if (!Array.isArray(numbers) || numbers.length < 1) {
```

Comment:
`Array.isArray` should be redundant here

Reply:
True, thanks for pointing it out, I've removed it